Dockerfile

```diff
@@ -22,7 +22,7 @@ RUN export CLOUD_SDK_REPO="cloud-sdk-$(lsb_release -c -s)" && \
     apt-get -y update && apt-get -y install google-cloud-sdk
 
 # Install kubectl
-ENV KUBECTL_VERSION 1.10.3
+ENV KUBECTL_VERSION 1.11.0
```

Comment:
Do we know exactly which version we need to be using, and is there a way to not hard-code this?